configure.ac

```diff
@@ -150,6 +150,13 @@
        echo ""
     fi
 
+    # Allow the user to specify what is the python shebang to use in the python
+    # scripts (can be useful when cross-compiling)
+    AC_ARG_WITH(python-shebang,
+           AS_HELP_STRING([--with-python-shebang], [Python shebang to use in python scripts]),
```

Comment:
I think I'd rather this be `--with-target-python`. Otherwise I don't see any issues here.